arm-iothub/2016-02-03/swagger/iothub.json

```
@@ -0,0 +1,1936 @@
+          "items": {
+            "$ref": "#/definitions/MetricDimension"
+          }
+        },
+        "availabilities": {
+          "description": "Gets or sets the metric availabilities.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MetricAvailability"
+          }
+        }
+      }
+    },
+    "LogSpecification": {
+      "description": "Contains properties related to a log specification.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or sets the name.",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Gets or sets the display name.",
+          "type": "string"
+        },
+        "blobDuration": {
+          "description": "Gets or sets the duration of the BLOB.",
+          "type": "string"
+        }
+      }
+    },
+    "MetricDimension": {
+      "description": "Contains properties related to a Metric Dimension.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or sets the name.",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Gets or sets the display name.",
+          "type": "string"
+        }
+      }
+    },
+    "MetricAvailability": {
+      "description": "Contains properties related to a Metric Availability.",
+      "type": "object",
+      "properties": {
+        "timeGrain": {
+          "description": "Gets or sets the time grain.",
+          "type": "string"
+        },
+        "blobDuration": {
+          "description": "Gets or sets the duration of the BLOB.",
+          "type": "string"
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "Contains properties related to the details of an error.",
+      "type": "object",
+      "properties": {
+        "Code": {
+          "description": "Gets or sets the code.",
+          "type": "string",
+          "x-ms-enum": {
```

its a by product of the order in which the massaging was being done in code.. Good catch :) 
So this is actually an enum in code so the swaggerschemafilter plugin adds the tag.. however at a later point there is code that removes the enum because we don't want to surface the specific error codes out.. Will fix this.